kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -172,10 +205,10 @@ namespace Kratos
 
 		// Compute the number of intersected edges
 		constexpr std::size_t n_edges = (TDim == 2) ? 3 : 6;
-		array_1d<unsigned int, n_edges> cut_edges_vector;
 		array_1d<double, n_edges> cut_edges_ratio_vector;
+		array_1d<double, n_edges> cut_extra_edges_ratio_vector;
 		std::vector<array_1d <double,3> > int_pts_vector;
-		const unsigned int n_cut_edges = ComputeEdgesIntersections(rElement1, rIntersectedObjects, cut_edges_vector, cut_edges_ratio_vector, int_pts_vector);
+		const unsigned int n_cut_edges = ComputeEdgesIntersections(rElement1, rIntersectedObjects, cut_edges_ratio_vector, cut_extra_edges_ratio_vector, int_pts_vector);
```

Comment:
These changes are only aesthetic right?

Reply:
This change is, because the cut_extra_edges_ratio_vector is needed for the calculation of the extrapolated edge ratio, which was not needed before. And instead of adding it, I replaced the cut_edges_vector, which was not used anymore.

Reply:
Perfect then.